lisa/transformers/dom0_kernel_installer.py

```diff
@@ -178,22 +178,22 @@ def _update_mariner_config(
     current_kernel: str,
     new_kernel: str,
 ) -> None:
-    ll_conf_file: str = "/boot/mariner-mshv.cfg"
+    mariner_config: str = "/boot/mariner-mshv.cfg"
```

Comment:
can you confirm with dev, is there a way to detect/read the expected names, instead of changing them every time when the file name changes